package.json

```diff
@@ -34,9 +30,9 @@
     "chai": "^4.1.2",
     "mocha": "^5.0.5",
     "nock": "^9.2.3",
-    "sinon": "^4.5.0",
+    "sinon": "^7.4.0",
```

Comment:
https://github.com/sinonjs/sinon/compare/v4.5.0...v7.4.1 - too big to realistically review but no indication of any issues ✅